miasm/analysis/data_flow.py

```diff
@@ -1908,9 +1908,9 @@ def __ne__(self, other):
         # required Python 2.7.14
         return not self == other
 
-    def may_interfer(self, dsts, src):
+    def may_interfere(self, dsts, src):
```

Comment:
I think for this function you should provide a wrapper with the old name (marked as deprecated). This ensures any usages won’t straight up break…